READMEs/data-nfts-and-datatokens-flow.md

```diff
@@ -31,6 +31,14 @@ docker system prune -a --volumes
 ./start_ocean.sh --with-c2d
 ```
 
+When running barge with c2d enabled, use the following bash script to wait until c2d is fully deployed:
```

Comment:
This script also waits until the `artifacts/` directory and `address.json` file are successfully created. I think the explanation should mention this as well.